internal/experiments/experiments.go

```diff
@@ -0,0 +1,21 @@
+package experiments
+
+type ExperimentConfig struct {
+	Name    string
+	Enabled bool
+}
+
+var exp = []ExperimentConfig{}
```

Comment:
Why use a slice if you need to iterate at lookup time?

Reply:
Will switch to map along with change suggested below on tolerating X% of nodes in a pool in an unknown state

Reply:
done in latest commits